### PR TITLE
Dedicated redis and postgresql for review app

### PR DESCRIPTION
**Pull request by vipin-dfe**
### Trello card
https://trello.com/c/kh03fNsN

### Context
The school experience review application uses a shared Redis and Postgresql instance. This change will allow a dedicated Redis and Postgresql pod to be deployed for every PR.

### Changes proposed in this pull request
The following changes are proposed as part of the change
- Allow dedicated Redis and Postgres
- Use an encoded database connection url
- Deprecates unused variables

### Guidance to review
All changes are only to allow for a dedicated Redis and Postgresql for review app.

### CODE REVIEW

**Comment by saliceti** [comment]
[School experience: dedicated redis and postgres for review apps](https://trello.com/c/kh03fNsN/2115-school-experience-dedicated-redis-and-postgres-for-review-apps)

**Comment by vipin-dfe** [comment]
Closing PR.

File: config/database.yml [hunk 1]
```diff
@@ -98,10 +101,10 @@ test: &test
 #
 production: &production
   <<: *default
-  host: <%= ENV['DB_HOST'] %>
-  database: <%= ENV['DB_DATABASE'] %>
-  username: <%= ENV['DB_USERNAME'] %>
-  password: <%= ENV['DB_PASSWORD'] %>
+  # host: <%= ENV['DB_HOST'] %>
```
Comment (saliceti) [inline config/database.yml:104]: Comments can be removed

File: terraform/aks/database.tf [hunk 2]
```diff
@@ -17,10 +17,12 @@ module "postgres" {
   azure_extensions            = ["POSTGIS", "address_standardizer", "plpgsql", "postgis_raster", "uuid-ossp", "citext"]
   azure_enable_high_availability = var.postgres_enable_high_availability
   azure_maintenance_window       = var.azure_maintenance_window
+  server_docker_image         = "postgis/postgis:14-3.4"
+  create_database = var.create_database
```
Comment (saliceti) [inline terraform/aks/database.tf:21]: Indentation

File: config/database.yml (hunk 1 above)
Comment (vipin-dfe) [reply config/database.yml]: applied.

File: terraform/aks/database.tf (hunk 2 above)
Comment (vipin-dfe) [reply terraform/aks/database.tf]: done.

**Comment by saliceti** [comment]
Test closing

File: docs/documents/release-process.md [hunk 3]
```diff
@@ -4,4 +4,26 @@
...
-1. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+2. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+
+## Configuring a school profile in a review app or local deployment for testing
+
+1. Ensure you have a login to DfE Sign In (pre-prod) with permission to manage any of the schools listed in `db/data/example_schools.csv`
+2. In a School Experience app running locally or as review app, click on [Manage school experience](/schools)
+3. Click on "Start now >" and you will be directed to login to DfE Sign In
+4. Login to DfE Sign In
+5. After signing in, you will be directed back to the Get Into Teaching website
```
Comment (saliceti) [inline docs/documents/release-process.md:15]: School experience website?

File: docs/documents/release-process.md [hunk 4]
```diff
@@ -4,4 +4,26 @@
    
    * When deploying a review application a script is run to find the first available static route (review-school-experience[1-20] ) and attach that to the review, via terraform. This enables the DFE Sign In process to work.
 
-1. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+2. Merging the PR will deploy the changes to the staging environment and, if successful, the production environment
+
+## Configuring a school profile in a review app or local deployment for testing
+
+1. Ensure you have a login to DfE Sign In (pre-prod) with permission to manage any of the schools listed in `db/data/example_schools.csv`
```
Comment (saliceti) [inline docs/documents/release-process.md:11]: Do we have a process for dfe sign in? We could at least point at their contact: https://technical-guidance.education.gov.uk/infrastructure/support/#dfe-sign-in

**Review by saliceti — APPROVED** [review]

**Pull request merged**